conduit/data/datamodules/audio/ecoacoustics.py

```diff
@@ -0,0 +1,99 @@
+"""Ecoacoustics data-module."""
+
+from typing import Any, Optional, Union
+
+import albumentations as A
+from kit import implements, parsable
+from kit.torch import prop_random_split
+from kit.torch.data import TrainingMode
+from pytorch_lightning import LightningDataModule
+
+from conduit.data.datamodules.base import CdtDataModule
+from conduit.data.datasets.audio.ecoacoustics import Ecoacoustics
+from conduit.data.structures import TrainValTestSplit
+
+from .base import CdtAudioDataModule
+
+__all__ = ["EcoacousticsDataModule"]
+
+
+class EcoacousticsDataModule(CdtAudioDataModule):
+    """Data-module for the Ecoacoustics dataset."""
+
+    @parsable
+    def __init__(
+        self,
+        root: str,
+        *,
+        image_size: int = 224,
+        train_batch_size: int = 32,
+        eval_batch_size: Optional[int] = 64,
+        num_workers: int = 0,
+        val_prop: float = 0.2,
+        test_prop: float = 0.2,
+        class_train_props: Optional[dict] = None,
+        seed: int = 47,
+        persist_workers: bool = False,
+        pin_memory: bool = True,
+        stratified_sampling: bool = False,
+        instance_weighting: bool = False,
+        training_mode: Union[TrainingMode, str] = "epoch",
+        specgram_segment_len: float = 15,
+        num_freq_bins: int = 120,
+    ) -> None:
+        super().__init__(
+            root=root,
+            train_batch_size=train_batch_size,
+            eval_batch_size=eval_batch_size,
+            num_workers=num_workers,
+            val_prop=val_prop,
+            test_prop=test_prop,
+            seed=seed,
+            persist_workers=persist_workers,
+            pin_memory=pin_memory,
+            stratified_sampling=stratified_sampling,
+            instance_weighting=instance_weighting,
+            training_mode=training_mode,
+        )
+        self.image_size = image_size
+        self.class_train_props = class_train_props
+        self.specgram_segment_len = specgram_segment_len
+        self.num_freq_bins = num_freq_bins
+
+    @property  # type: ignore[misc]
+    @implements(CdtAudioDataModule)
+    def _base_augmentations(self) -> A.Compose:
```

Comment:
these augmentation properties were removed a couple of weeks ago and replaced with default_transforms, with the transforms now able to be freely specified through the init.